src/main/java/io/r2dbc/postgresql/api/PostgresqlConnection.java

```diff
@@ -170,4 +172,13 @@
     @Override
     Mono<Boolean> validate(ValidationDepth depth);
 
+    /**
+     * Copy bulk data from client into a PostgreSQL table very fast.
```

Comment:
Nit: We should mention `COPY … FROM STDIN` here as well.

Reply:
I did copy it from pgjdbc's CopyManager 😄